src/diffusers/schedulers/scheduling_heun_discrete.py

```diff
@@ -106,6 +112,8 @@ def __init__(
         elif beta_schedule == "squaredcos_cap_v2":
             # Glide cosine schedule
             self.betas = betas_for_alpha_bar(num_train_timesteps)
+        elif beta_schedule == "exp":
+            self.betas = betas_for_alpha_bar(num_train_timesteps, alpha_bar_fn=lambda t: math.exp(t * -12.0))
```

Comment:
Let's try to not pass functions here. Can you instead pass something like `alpha_transform_type="exp"`